man/docker-info.1.md

```diff
@@ -11,9 +11,9 @@ docker-info - Display system-wide information
 
 # DESCRIPTION
 This command displays system wide information regarding the Docker installation.
-Information displayed includes the number of containers and images, pool name,
-data file, metadata file, data space used, total data space, metadata space used
-, total metadata space, execution driver, and the kernel version.
+Information displayed includes the number of containers and (non-tagged) images,
```

Comment:
in that case, it would be better if the `docker info` output said so

something like

```
12:44 $ docker info
Containers: 62
 Running: 0
 Paused: 0
 Stopped: 62
Untagged Images: 662
Server Version: 1.10.3
...
```